### PR TITLE
PW5311 Replace secured fields enabled with is Adyen

### PR DESCRIPTION
**Pull request by zaiddreakh**
## Tested scenarios

**Fixed issue**:

### CODE REVIEW

File: src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/payment_instruments/savePayment.js [hunk 1]
```diff
@@ -44,10 +44,10 @@ function getResponseBody(action) {
 }
 
 function savePayment(req, res, next) {
-  if (!AdyenHelper.getAdyenSecuredFieldsEnabled()) {
-    return next();
+  if([constants.METHOD_ADYEN_POS, constants.METHOD_ADYEN_COMPONENT, 'ADYEN_CREDIT']
```
Comment (maassenbas) [inline src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/payment_instruments/savePayment.js:47]: ADYEN_POS is not necessary here

**Review by maassenbas — APPROVED** [review]

File: src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/payment_instruments/savePayment.js [hunk 2]
```diff
@@ -44,10 +44,13 @@ function getResponseBody(action) {
 }
 
 function savePayment(req, res, next) {
-  if (!AdyenHelper.getAdyenSecuredFieldsEnabled()) {
+  if (
+    [constants.METHOD_ADYEN_COMPONENT, 'ADYEN_CREDIT'].indexOf(
```
Comment (wboereboom) [inline src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/payment_instruments/savePayment.js:48]: Is there no constant we can declare for ADYEN_CREDIT?

**Review by wboereboom — APPROVED** [review]
🇯🇴

File: src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/payment_instruments/savePayment.js (hunk 2 above)
Comment (zaiddreakh) [reply src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/payment_instruments/savePayment.js:48]: no, and actually I just noticed `constants.METHOD_ADYEN_COMPONENT` returns "AdyenComponent" whereas the expected payment processor ID is "Adyen_Component" so I can't use the constants here

**Review by wboereboom — APPROVED** [review]
Elegant fix for the hard-coded values. 
LGTM
🦆

**Review by maassenbas — APPROVED** [review]

**Pull request merged**